File: backend/GymTurf.Domain/Entities/Branch.cs
using System;
using GymTurf.Domain.Common;

namespace GymTurf.Domain.Entities;

public class Branch : BaseEntity
{
    public string Name { get; set; } = string.Empty;
    public string Address { get; set; } = string.Empty;
    public string PhoneNumber { get; set; } = string.Empty;
    public bool IsActive { get; set; } = true;
}

File: frontend/src/types/branch.ts
export interface Branch {
  id: string;
  name: string;
  address: string;
  phoneNumber?: string;
  isActive: boolean;
}

export interface Facility {
  id: string;
  branchId: string;
  name: string;
  type: string; // Gym, Turf, Court
  capacity: number;
  pricePerHour: number;
  isActive: boolean;
}
